pkg/crd/markers/patch_validation.go

```diff
@@ -0,0 +1,16 @@
+package markers
+
+import "sigs.k8s.io/controller-tools/pkg/markers"
+
+const OpenShiftFeatureSetMarkerName = "openshift:enable:featureSet"
```

Comment:
Kubebuilder uses `PascalCase` for the third element, I think we should be consistent 
```suggestion
const OpenShiftFeatureSetMarkerName = "openshift:enable:FeatureSet"
```

Reply:
Ah, coding via side-effect.  change this case and we end up down a different code path that fails in controller-tools.  Such a pain.  I'll chase it and prepare for sadness.